item.ts

```diff
@@ -86,4 +86,37 @@ export type RequiredUniqueTextItem = UniqueTextItem & RequiredSingleAnswer;
 export interface ConstrainedListItemValue extends FormItem {
   readonly text: string;
   readonly code?: string | number;
+  readonly system?: string | null;
+  readonly label?: string | null;
+  readonly score?: string | null | number;
 }
+
+export interface ContactAddressItem extends FormItem {
+  readonly items: [
+    houseOrBuilding,
+    townOrCity,
+    stateOrProvince,
+    zipOrPostal,
+    CountryOrRegion,
+  ];
+}
+export interface houseOrBuilding extends FormItem {
```

Comment:
Please fix the case - `houseOrBuilding` is not correct it should be `HouseOrBuilding`. Please fix all the others as well.

Reply:
Fixed the case in interfaces and updated